src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -136,6 +136,9 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="-static-pie" Condition="'$(StaticExecutable)' == 'true' and '$(PositionIndependentExecutable)' != 'false'" />
       <LinkerArg Include="-dynamiclib" Condition="'$(_IsApplePlatform)' == 'true' and '$(NativeLib)' == 'Shared'" />
       <LinkerArg Include="-shared" Condition="'$(_IsApplePlatform)' != 'true' and '$(NativeLib)' == 'Shared'" />
+      <!-- source-build links directly with openssl (-lssl -lcrypto) -->
+      <LinkerArg Include="-lssl" />
+      <LinkerArg Include="-lcrypto" />
```

Comment:
This relates to https://github.com/dotnet/runtime/issues/66859.

I've unconditionally added these flags because there is no condition that tells us if the build links with OpenSSL (source-build non-portable) or loads it dynamically (`dlopen`).

In the latter case the flags are not needed. Perhaps they don't cause issues either.